test/verify/check-storage-smart

```diff
@@ -66,10 +66,10 @@ class TestStorageSmart(storagelib.StorageSmartCase):
         else:
             return False
 
-    def testSmart(self):
-        def set_smart_dump(name: str, block: str):
-            self.machine.execute(f"udisksctl smart-simulate -f /tmp/smart-dumps/{name} -b {block}")
+    def set_smart_dump(self, name: str, block: str):
+        self.machine.execute(f"udisksctl smart-simulate -f /tmp/smart-dumps/{name} -b {block}")
```

Comment:
Still only used in `testSmart` right?

Reply:
yah, at first I had this in two tests but I moved it back to one so it doesn't need to be here but in the end it shouldn't matter all that much